test-e2e/codecept.mobile.js

```diff
@@ -12,7 +12,7 @@ exports.config = {
   grep: '(?=.*)^(?!.*@desktop-only)',
   tests: ['./tests/**/*.js', './tests/**/*.ts'],
   output: './output/mobile',
-  timeout: 60,
+  timeout: 3000,
```

Comment:
I believe this value is in seconds, so 3000 is almost an hour. We probably don't want to have tests that hang for that long. https://codecept.io/configuration/